src/run/run.c

```diff
@@ -2342,6 +2384,7 @@ static int start_transient_scope(sd_bus *bus) {
         }
 
         (void) polkit_agent_open_if_enabled(arg_transport, arg_ask_password);
+        (void) ask_password_agent_open_if_enabled(arg_transport, arg_ask_password);
```

Comment:
I don't see the point of spawning ask password agent for scope and trigger units? They don't execute anything before we exit/exec...

Reply:
they might, absolutely, they block for the start job to be executed via bus_wait_for_jobs_one(), and they might dep on pretty much anything that might ask questions like this, you never know. it might be as trivial as `-p After=unlockmycryptdevice` or so...